app-api/src/main/java/org/openmrs/eip/app/management/entity/SenderSyncResponse.java

```diff
@@ -0,0 +1,49 @@
+package org.openmrs.eip.app.management.entity;
+
+import java.util.Date;
+
+import javax.persistence.Access;
+import javax.persistence.AccessType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "sender_sync_response")
+public class SenderSyncResponse extends AbstractEntity {
+	
+	public static final long serialVersionUID = 1;
+	
+	@NotNull
+	@Column(name = "message_uuid", length = 38, nullable = false, updatable = false)
+	@Access(AccessType.FIELD)
+	private String messageUuid;
+	
+	@NotNull
+	@Column(name = "date_sent", nullable = false, updatable = false)
+	@Access(AccessType.FIELD)
+	private Date dateSent;
```

Comment:
Same, here why is the access type set to field? This field has a public setter, I think you can remove the @access annotation

Reply:
The explanation is the same as [here](https://github.com/FriendsInGlobalHealth/openmrs-eip/pull/11#discussion_r909244325).

I'll remove it.